src/components/CFloatingButton/index.meta.js

```diff
@@ -0,0 +1,108 @@
+import { merge } from 'lodash';
+import CButton from '../CButton/index.meta';
+
+export default {
+  group: 'actions',
+  type: 'floating-button',
+  name: 'Floating Button',
+  icon: 'control_point',
+  children: [
+    'floating-button-item',
+  ],
+  actions: merge({}, CButton.actions),
+  events: merge({}, CButton.events),
+  options: {
+    color: true,
+    theme: true,
+    itemsCount: {
+      type: 'childrenCountInput',
+      name: 'Number of Child Elements',
+      value: '0',
+      validation: {
+        required: true,
+        min: 0,
+      },
+      priority: 3,
+    },
+    openOnHover: {
+      type: 'check',
+      name: 'Open On Hover',
+      value: true,
+      priority: 4,
+    },
+    absolute: {
+      type: 'check',
+      name: 'Position The Element Absolutely',
+      value: false,
+      priority: 5,
+    },
+    fixed: {
+      type: 'check',
+      name: 'Position The Element Fixed',
+      value: false,
+      priority: 6,
+    },
+    direction: {
+      type: 'select',
+      name: 'Direction In Which Content Will Be Shown',
+      value: 'top',
+      items: [
+        {
+          name: 'Left',
+          value: 'left',
+        },
+        {
+          name: 'Right',
+          value: 'right',
+        },
+        {
+          name: 'Bottom',
+          value: 'bottom',
+        },
+        {
+          name: 'Top',
+          value: 'top',
+        },
+      ],
+      priority: 7,
+    },
+    align: {
```

Comment:
Not sure is this supposed to define button position? I see it gets assigned to `align` prop, but I don't see that prop in Vuetify specs for v-speed-dial 😕 And it looks as if it has no effect.